src/components/Field.vue

```diff
@@ -62,15 +62,15 @@
             <div v-bind:key="optionIndex" v-for="(option, optionIndex) in options" class="radio radio-icon">
               <input :name="fieldName" :id="fieldName + '_' + optionIndex" type="radio" :value="option.value" v-model="value">
               <label :for="fieldName + '_' + optionIndex">
-                <span class="check"><i class="fa fa-circle"></i></span> {{ option.label || option.value }}
+                <span class="check"><i class="fa fa-circle"></i></span> <span v-if="option.label" v-html="option.label"></span><template v-else>{{ option.value }}</template>
```

Comment:
[MINOR] @tonytlwu, from a readability perspective, it's preferred to have v-if and v-else in separate lines


```suggestion
                <span class="check"><i class="fa fa-circle"></i></span> 
                <span v-if="option.label" v-html="option.label"></span>
                <template v-else>{{ option.value }}</template>
```

Reply:
Updates commited